docs/installation/install-livepeer/binary-release.md

```diff
@@ -32,6 +32,9 @@ tar -zxvf livepeer-linux-amd64.tar.gz
 
 # Finally, move it to the appropriate directory
 mv livepeer-linux-amd64/* /usr/local/bin/
+
+# Currently, packages are available for Arch Linux
+paru go-livepeer-bin
```

Comment:
what does this command do exactly?

Reply:
It installs the [livepeer-go-bin](https://aur.archlinux.org/packages/go-livepeer-bin/) package from the [Arch User Repository (AUR)](https://aur.archlinux.org/).

`Pacman` is the package management tool used by Arch Linux, akin to `apt` on Debian / Ubuntu. However it will only install packages from the `core` and `community` repositories. All other packages for Arch are hosted on the AUR.
The only way Livepeer, or any other non-core package, may be included in the community repo on Arch Linux is if enough users upvote it on the AUR.

Pacman does not install packages from the AUR, the official way to install AUR packages is to build each package manually `makepkg -si` which isn't very user friendly if you install more than a few packages since it does no tracking of installed packages.
The alternative is a setup a repository which is _not_ recommended by the Arch Linux developers as hosting via the AUR let's Arch developers examine the package source files for things like malicious injection etc.

So [there are tools](https://wiki.archlinux.org/title/AUR_helpers) built to manage AUR packages, [Paru](https://aur.archlinux.org/packages/paru), is the most popular tool with users new to Arch Linux, I personally use `aurutils`. If any experienced Arch user sees `paru`, they know it's an AUR package, majority of new users would default to using `paru` anyways, which is why I used that as the most obvious example for Arch Linux.

I hope that clear things up.

See:
1) https://wiki.archlinux.org/title/Arch_User_Repository
2) https://wiki.archlinux.org/title/AUR_helpers#Comparison_tables
3) https://wiki.archlinux.org/title/Pacman#Repositories_and_mirrors